src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -201,6 +201,13 @@ export default class ReviewCollapsibleChapter extends React.Component {
                     ) : (
                       <ProgressButton
                         submitButton
+                        onButtonClick={() => {
+                          focusOnChange(
+                            `${page.pageKey}${
+                              typeof page.index === 'number' ? page.index : ''
+                            }`,
+                          );
+                        }}
```

Comment:
Also, if we don't have to depend on `tabindex` being set dynamically, could `focusElement` be called here directly if the right node could be found?

Reply:
See above comment about waiting for DOM to render. Should this action be moved into the SchemaForm `onChange` property, to get around render delay?

Reply:
This PR no longer modifies the tabindex. Focus is returned to the "Edit" button.

Unit tests have also been added.

Reply:
We're getting close now! We need to set focus on the `<p>` tag after users press Edit. Might need a `tabIndex="-1"` to capture that. Also, can we scroll up a tad further when users press the Update button? The edit button is halfway out of the viewport.

---
![Screen Shot 2020-03-12 at 3 03 01 PM](https://user-images.githubusercontent.com/934879/76562237-f2285400-6472-11ea-8540-144813934f53.png)

Reply:
Oh, I missed this... I thought I had it covered in the upcoming PR, but this happens when you click the edit button directly.

But, some forms don't have `<p>` tags (array types like contested issues in form 526), so how about I target either the associated `<legend>` or `<label>`? Should those also have the outline hidden?

Reply:
Yes, those should also have the outline hidden.

Reply:
I also had to modify the scroller to get the offset to work properly, so now the edit button should be in view! 🎉

Reply:
Ok, the `<legend>` and `<label>` outlines have been removed as well.